docs/csharp/getting-started/index.md

```diff
@@ -30,6 +30,6 @@ The following topics are available:
 
 - [Getting started with Visual Studio Code](with-visual-studio-code.md)
 
-   Visual Studio Code is a programming editor for Windows, Linux, and macOS that supports IntelliSense (code completion) and debugging.
+   Visual Studio Code is a code editor redefined and optimized for building and debugging modern web and cloud applications. Visual Studio Code is free and available on your favorite platform - Linux, Mac OSX, and Windows.
```

Comment:
I see this comes from the marketing site description. I like it but would prefer if we leave the marketing tone out of it. 

It could be rewritten similar to this:
"Visual Studio Code is a free code editor optimized for building and debugging modern web and cloud applications. It supports IntelliSense and is available for Linux, macOS, and Windows."